### DISCUSSION TITLE
Использование списка сокращений

### DISCUSSION THREAD

**Discussion by pavel-vash** [rominf/latex-g7-32#135] [Q&A]
Скажите, а список используемых сокращений только у меня не работает?

\documentclass{G7-32}
\begin{document}
\printnomenclature
Текст
\nomenclature{АДХ}{Аэродинамические характеристики}
\end{document}

После pdflatex получается PDF с единственным словом "Текст"

**Comment by einhander** [comment]
Используйте дополнительный шаг при сборке:
````
makeindex <filename>.nlo -s nomencl.ist -o <filename>.nls
````

**Reply by pavel-vash** [reply]
Спасибо. это сработало, если \documentclass{article}
Когда включаю \documentclass{G7-32}, то после pdflatex и makeindex повторный pdflatex выдает ошибку:

! LaTeX Error: Something's wrong--perhaps a missing \item.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.4   \item [{АДХ}]
                      \begingroup Аэродинамические хара�...

**Reply by einhander** [reply]
Попробуйте собрать полностью пример из шаблона, раздел термины и определения допиливался чтобы работало.

**Reply by pavel-vash** [reply]
Пример шаблона сработал, но я обнаружил проблему в том, что если есть только 
\Abbrev{}{} , то при сборке происходит сбой.
Для того, чтобы сбоя не было, нужно еще и использовать 
\Define{}{}
То есть если не включены "Термины и определения", то не получается использовать "Перечень сокращений и обозначений"
